packages/core/src/App/Containers/Routes/routes.jsx

```diff
@@ -18,65 +18,66 @@ const Error = Loadable({
     },
 });
 
-class Routes extends React.Component {
-    unlisten_to_change = null;
-    initial_route = null;
+const Routes = ({ error, has_error, history, is_logged_in, location, passthrough, ...props }) => {
+    let unlisten_to_change = null;
+    let initial_route = null;
```

Comment:
is it possible to use React.useState instead of let here?

Reply:
I change them to `Ref`, since I don't want the component rerender after their change.